starter/script.js

```diff
@@ -1 +1,74 @@
+var userName = prompt("Please type your first name!");
+//Attempt to weed out incorrect inputs- works for numbers but not for symbols!
+if (isNaN(userName) === false){document.write("Oops, that's not a valid name! Refresh your browser and try again!")}
+else {
+    //Question 2: Prompt; Recipe or Workout plan
+    var fitOrFood = prompt("Greetings, " + userName + "!\nType \[H\] to select a healthy smoothie recipe -or- type \[W\] for a workout plan")
+
+    //Question 3a: Healthy Smoothie Recipe Options
+    if (fitOrFood == "H" || fitOrFood == "h"){var recipeChoice = prompt("Great choice! Does anything look appealing? \nPlease type the number that corresponds with your choice.\n\[1\] Ginger Mango Chai\n\[2\] Green Strawberry\-Kiwi Lemonade\n\[3\] Chocolate\-Covered Cherry Smoothie\n\[4\] Minty Fresh Green Smoothie")
+        if(Number(recipeChoice) === 1){document.write("<p  class=\"title\">" + "Ginger Mango Chai" + "<p>1 cup frozen mango<p> 1\/2 cup freshly brewed chai tea \(chilled\)<p>1 tsp freshly grated ginger<p>1 frozen banana<p>1\/4 cup coconut milk<p>1 tsp cinnamon<p  class=\"subtitle\">Blend ingredients in a high speed blender and enjoy!")}
+        else if(Number(recipeChoice) === 2){document.write("<p  class=\"title\">" + "Green Strawberry\-Kiwi Lemonade" + "<p>1 kiwi \(peeled\)<p> 3\/4 cup frozen strawberries \(chilled\)<p>1\/2 cup frozen pineapple<p>1\/2 lemon \(peel and seeds removed\)<p>2 cups spinach<p  class=\"subtitle\">Blend ingredients in a high speed blender and enjoy!")}
+        else if(Number(recipeChoice) === 3){document.write("<p  class=\"title\">" + "Chocolate\-Covered Cherry Smoothie" + "<p>1 cup almond milk \(or your favorite nut milk\)<p> 1 cup frozen cherries<p>1 tbsp cacao powder<p>3 cups spinach<p  class=\"subtitle\">Blend ingredients in a high speed blender and enjoy!")}
+        else if(Number(recipeChoice) === 4){document.write("<p  class=\"title\">" + "Minty Fresh Green Smoothie" + "<p>2 cups coconut water<p> 3 stalks kale<p>1 green apple \(cored\)<p>1\/2 cucumber<p>1\/2 cup mint leaves<p>2 dates \(pitted\)<p  class=\"subtitle\">Blend ingredients in a high speed blender and enjoy!")}
+        else{document.write("Oops, that's not a valid choice! Refresh your browser and try again!")}}
+
+    //Question 3b: Workout Plans
+    else if (fitOrFood == "W" || fitOrFood == "w"){
+      //Question 3b; HIIT Track
+      var workout = parseInt(prompt("Choose your workout style:\nPlease type the number that corresponds with your answer.\n\[1\] HIIT  \n\[2\] Yoga"));
+        if(Number(workout) === 1){
+          var hiitNameArray = userName.split("");
+          var i=0;
+          document.write("<p  class=\"title\">" + "Hey, "+ userName + "!" + "<p class=\"subtitle\">" + "Here's your customized workout, based on the letters in your name." + "<p>");
+              while(i < hiitNameArray.length) {
+                  if(hiitNameArray[i] === "A" || hiitNameArray[i] === "a"){document.write("50 jumping jacks")}
```

Comment:
What are some ways we can start DRY up this code? One way is to lower case `hiitNameArray[i]`. Then all we have to do is write a condition for the lower cased version. 

Additionally we recently learned about objects, how can we use an object instead of all these if/else conditionals?